2-mandatory/4-restaurant.js

```diff
@@ -21,6 +21,18 @@ const MENU = {
 
 let cashRegister = {
   // write code here
+  orderBurger : function (balance){
+                  if (balance>=MENU.burger){
+                    return balance = balance - MENU.burger
```

Comment:
you can also write: `return balance -= MENU.burger

Reply:
super thank you